src/script_opt/Stmt.cc

```diff
@@ -724,7 +713,7 @@ void StmtList::Inline(Inliner* inl)
 
 bool StmtList::IsReduced(Reducer* c) const
 	{
-	int n = Stmts().length();
+	auto n = Stmts().size();
```

Comment:
Why does `StmtList` call it's own `Stmts()` method if it has direct access to the object that method is returning already? This is kind of a general question across all of the code in `StmtList`.